clang/docs/ReleaseNotes.rst

```diff
@@ -214,6 +214,10 @@ New Compiler Flags
   This diagnostic can be disabled to make ``-Wmissing-field-initializers`` behave
   like it did before Clang 18.x. Fixes (`#56628 <https://github.com/llvm/llvm-project/issues/68933>`_)
 
+- ``-fexperimental-late-parse-attributes`` enables an experimental feature to
```

Comment:
@AaronBallman I added some documentation on the new flag. It's a bit vague right now but I'll make it more specific when the first use of `LateAttrParseExperimentalExt` is added.